packages/code.rb

```diff
@@ -6,11 +6,13 @@ class Code < Package
   version '1.25'
```

Comment:
Can you please bump the version number to 1.26? We need to update the already installed packages.

Reply:
Newest stable version is 1.26.1 now it seems.